src/sentry/web/frontend/debug/debug_notify_disable.py

```diff
@@ -0,0 +1,41 @@
+from django.http import HttpRequest, HttpResponse
+from django.views.generic import View
+
+from sentry import integrations
+from sentry.integrations.notify_disable import get_provider_type, get_url
+from sentry.models import Integration
+from sentry.testutils import TestCase
+
+from .mail import MailPreview
+
+
+class DebugNotifyDisableView(View, TestCase):
+    def get(self, request: HttpRequest) -> HttpResponse:
+        self.integration = Integration.objects.create(
+            provider="slack",
+            name="Awesome Team",
+            external_id="TXXXXXXXZ",
+            metadata={
+                "access_token": "xoxb-xxxxxxxxx-xxxxxxxxxx-xxxxxxxxxxxx",
+                "installation_type": "born_as_bot",
+            },
+        )
+
+        self.organization = self.create_organization()
```

Comment:
Instead of subclassing `TestCase` to use this you can just do it like this: https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/debug/debug_new_user_feedback_email.py#L13

That'll clean up the mypy error